src/dev/impl/DevToys/Views/Tools/Generators/PasswordGenerator/PasswordGeneratorPage.xaml

```diff
@@ -0,0 +1,132 @@
+﻿<Page
+    x:Class="DevToys.Views.Tools.PasswordGenerator.PasswordGeneratorPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:ex="using:DevToys.UI.Extensions"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required">
+
+    <Grid Margin="0,0,0,16" RowSpacing="12">
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <ex:IsCompactOverlayModeTrigger TargetElement="{x:Bind}" />
+                    </VisualState.StateTriggers>
+                    <VisualState.Setters>
+                        <Setter Target="LengthSetting.Visibility" Value="Collapsed" />
```

Comment:
This is great, but it doesn't hide the `Configuration` text.
I'd suggest to give a name to the StackPanel on line 42 and collapse the whole StackPanel  on line 21 (and remove lines 22 to 24).

Reply:
I believe I made the modification desired here.